style/pi-hole.css

```diff
@@ -115,3 +115,7 @@ a.lookatme {
     -webkit-animation: fa-spin 1s infinite linear;
     animation: fa-spin 1s infinite linear;
 }
+
+#MAC {
+    text-transform:lowercase;
+}
```

Comment:
You should create a CSS class instead of giving the same ID to multiple elements.

Reply:
Please see above. Let me know if you prefer custom classes, and I can do that! :)